compiler/rustc_codegen_ssa/src/mir/statement.rs

```diff
@@ -115,6 +115,29 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 self.codegen_coverage(&mut bx, coverage.clone());
                 bx
             }
+            mir::StatementKind::CopyNonOverlapping(box mir::CopyNonOverlapping {
+                ref src,
+                ref dst,
+                ref count,
+            }) => {
+                let dst_val = self.codegen_operand(&mut bx, dst);
+                let src_val = self.codegen_operand(&mut bx, src);
+                let count_val = self.codegen_operand(&mut bx, count);
+                let count = count_val.immediate_or_packed_pair(&mut bx);
+                let dst = dst_val.immediate_or_packed_pair(&mut bx);
+                let src = src_val.immediate_or_packed_pair(&mut bx);
+                use crate::MemFlags;
+                let flags = MemFlags::empty();
```

Comment:
Style nit: either `use` at the top of the module or `let flags = crate::MemFlags::empty()` – the separate inline `use` looks super jarring.